.circleci/config.yml

```diff
@@ -39,7 +39,6 @@ jobs:
           ./emsdk activate latest
           popd
           echo EMSCRIPTEN_ROOT="'~/emscripten/'" >> .emscripten
-          echo BINARYEN_ROOT="''" >> .emscripten
```

Comment:
The reason this was here is to make sure we use the binaryen port, at the current version the port has defined. That let us synchronize the two repos. What does this change do that that?

My guess would be that the appended line here was overriding the default from `settings_template_readonly.py`,
```
BINARYEN_ROOT = os.path.expanduser(os.getenv('BINARYEN', '')) # if not set, we will use it from ports
```
So it may not actually change anything?

Reply:
The .emscripten file in this case is coming from the emsdk.. which I had thought didn't set binaryen.  But on further investigation I now see that it does actually include binaryen.

I'll revert this part.

The good news is the emsdk already has binaryen in it.